smoke-tests/src/main/groovy/io/opentelemetry/smoketest/AbstractServerSmokeTest.groovy

```diff
@@ -17,7 +17,12 @@ abstract class AbstractServerSmokeTest extends AbstractSmokeTest {
   protected OkHttpClient client = OkHttpUtils.client()
 
   def setupSpec() {
-    PortUtils.waitForPortToOpen(httpPort, 240, TimeUnit.SECONDS, serverProcess)
+    try {
+      PortUtils.waitForPortToOpen(httpPort, 240, TimeUnit.SECONDS, serverProcess)
+    } catch (e) {
+      System.err.println(logfile.text)
```

Comment:
was this exception not getting logged anywhere? maybe `e.printStackTrace()` instead?

Reply:
The exception is re-thrown right after and handled correctly by the testing framework. The println is to show what went wrong in the child process. This is just to make debugging a bit easier.

Reply:
oh, this dumps the full text from the log file to stderr before the exception is logged by testing framework? got it, that makes sense